documentation/basics/headless.md

```diff
@@ -45,6 +45,7 @@ Windows:
 - Except for options passed with the `--export_source` argument (see below), don't specify any values with space-characters in them, even if inside quotes (e.g. `"my file name.h5"`, or `"(10, 20)"`). Due to a [bug in cpython](https://bugs.python.org/issue22433) those might be misinterpreted.
 - For paths to hdf5 datasests (either input or output), ilastik uses the same conventions as the generic [h5ls](https://support.hdfgroup.org/HDF5/Tutor/cmdtoolview.html#h5ls) utility. That is, the hdf5 dataset name should be appended to the file path: `/path/to/my_file.h5/internal/path/to/dataset`.
 - In the current "headless" implementation of object classification, the entire image is loaded into RAM in one go, and then object classification is run on it.  Therefore, there is a limit to how large your input image can be.
+- ilastik's command line options use underlines rather than dashes to separate words (e.g.: `--cutout_subregion` and not `--cutout-subregion`). If you mix them up, you might get strange errors. See [Known Issues](#known-issues) for more info.
```

Comment:
The _argparse_ module actually converts flags defined as `--cutout-subregion` into `args.cutout_subregion`, and I'm positive we can modify _ArgumentParser_ to accept either `-` or `_` as separators.

Reply:
Yup. We could do something like `parser.add_argument('--my_flag', '--my-flag', ...)` everywhere we create arguments, but we'd have to do that even for the arguments which are created dynamically and get that merged before the next release. And still there will be people stuck on older versions, so we might as well have that documented somewhere, I guess.

But I agree. Lets fix it =D